packages/orbit-components/src/Hide/Hide.stories.tsx

```diff
@@ -1,70 +1,87 @@
 import * as React from "react";
+import type { Meta, StoryObj } from "@storybook/react";
 
 import Separator from "../Separator";
 import ChevronBackward from "../icons/ChevronBackward";
 import type { Devices } from "../utils/mediaQuery";
 
 import Hide from ".";
 
-export default {
-  title: "Hide",
+type HidePropsAndCustomArgs = React.ComponentProps<typeof Hide> & {
+  largeDesktop: boolean;
+  desktop: boolean;
+  tablet: boolean;
+  largeMobile: boolean;
+  mediumMobile: boolean;
+  smallMobile: boolean;
 };
 
-export const WithSeparator = ({ block }) => {
-  return (
-    <Hide on={["largeMobile"]} block={block}>
-      <Separator />
-    </Hide>
-  );
+const meta: Meta<HidePropsAndCustomArgs> = {
+  title: "Hide",
+  component: Hide,
+  parameters: {
+    info: "Check Orbit.Kiwi for more detailed design guidelines.",
+    controls: {
+      exclude: ["on"],
+    },
+  },
+
+  args: {
+    largeDesktop: false,
+    desktop: false,
+    tablet: false,
+    largeMobile: false,
+    mediumMobile: false,
+    smallMobile: false,
+    block: false,
+  },
 };
 
-WithSeparator.story = {
+export default meta;
+
+type Story = StoryObj<HidePropsAndCustomArgs>;
+
+export const WithSeparator: Story = {
+  render: ({ largeMobile, ...args }) => {
+    const on = [largeMobile && "largeMobile"] as Devices[];
+
+    return (
+      <Hide {...args} on={on}>
+        <Separator />
+      </Hide>
+    );
+  },
```

Comment:
In this story we have a bunch of controls but they don't seem to work.

<img width="529" alt="image" src="https://github.com/user-attachments/assets/f2ed5636-faf8-4e45-bc2e-7de584fb32b5">


Perhaps we should exclude them and only keep them for the Playground?

Reply:
Yes, it totally makes sense to keep only `Playground` story.
Also, I have found that `Separator` component is not the best representative for this type of visualization, so I'll replace it with `div` with background color to have the actual state of component more highlighted. :D 

something like this:
<img width="586" alt="image" src="https://github.com/user-attachments/assets/d2e026e3-844c-488a-8a85-410e2fa994a2">

WDYT?

Reply:
Ok, it makes sense.